src/App.vue

```diff
@@ -161,6 +162,23 @@ export default {
       });
       return;
     },
+    /**
+     * @param {String} newPredator is value of new predator type
+     * we generate list of allowed disciplines based on it
+     * we will check if current selected predator discipline is allowed and if not, we'll reset it
+     * then we need to update list of allowed disciplines 
+     * and then we can set new predator type value
+     */
+    predatorTypeChange(newPredator) {
```

Comment:
urcite to kontrolovat v app? ja by som to asi urobil trocha inak, nezda sa mi to...
mozno pozri v tom vue navode ako riesia event validaciu, uz si to nepametam

Reply:
njn, zda sa logictejsie to riesit nizsie. Dneska som zkusil tri rozne zposoby ako to urobit, ale ziadny mi uplne nesiel alebo nefungoval tak ako by som chcel. Ak na tom trvas navrhujem vytvorit novy task, kde sa to urobi.

Reply:
a dnes sa mi to podarilo :-)